js/rpg_core/Graphics.js

```diff
@@ -1228,8 +1228,8 @@ Graphics._requestFullScreen = function() {
  * @private
  */
 Graphics._cancelFullScreen = function() {
-    if (document.cancelFullScreen) {
-        document.cancelFullScreen();
+    if (document.fullscreenElement) { 
```

Comment:
Thanks!

Could you fix this line to below?
`if (document.exitFullscreen) {`

Reply:
Ok. I changed it to:
```

 if (document.exitFullscreen) { 
        document.exitFullscreen();
```
based on https://www.sitepoint.com/use-html5-full-screen-api